src/components/views/donate/CryptoDonation.tsx

```diff
@@ -348,6 +361,8 @@ const CryptoDonation = (props: {
 					setFailedModalType={setFailedModalType}
 					setTxHash={setTxHash}
 					project={project}
+					mainProjectAddress={mainProjectAddress}
+					secondaryProjectAdress={secondaryProjectAddress}
```

Comment:
we never use these addresses in the donate modal. I think we can remove them.

Reply:
we need it for the confirmDonation method :)